ansys/api/additive/v0/additive_domain.proto

```diff
@@ -30,7 +30,7 @@ message MeltPoolTimeStep {
 
 message MeltPool {
     repeated MeltPoolTimeStep time_steps = 1;
-    bytes thermal_history_vtk = 2;
+    string thermal_history_vtk_zip = 2;
```

Comment:
By changing the type and name of field 2, you have a breaking change. This should probably be
```
message MeltPool {
    reserved 2;
    repeated MeltPoolTimeStep time_steps = 1;
    string thermal_history_vtk_zip = 3;
```
That would be the "best practices" way to update this message. However, since we're the only ones using this API, what you have will work.